src/server/controllers/match.controller.js

```diff
@@ -0,0 +1,127 @@
+'use strict';
+
+const matchSchema = require('mongoose').model('match');
+import mailer from 'nodemailer';
+
+/*
+ * Methods about mentoring request, accept or reject including E-mail Service
+ */
+
+let matchData = new matchSchema();
+
+let sampleSuccessResult = {
+  successCode: 1,
+  matchData: matchData,
+  requestList: {
+    accepted: [
+      {
+        mentorId: 'abc123abc123',
+        mentorName: 'Jung Su Kim',
+        mentorPic: 'http://789ghi012jkl.jpg',
+      },
+    ],
+    requested: [
+      {
+        menteeId: 'abc456abc456',
+        menteeName: 'Marry Su',
+        menteePic: 'http://123abc456def.png',
+      },
+      {
+        menteeId: 'aaa111aaa111',
+        menteeName: 'Jane Park',
+        menteePic: 'http://123abc456def.png',
+      },
+    ],
+    pending: [
+      {
+        mentorId: 'aaa222aaa222',
+        mentorName: 'Sun Lee',
+        mentorPic: 'http://789ghi012jkl.jpg',
+      },
+      {
+        mentorId: 'aaa333aaa333',
+        mentorName: 'Harry Potter',
+        mentorPic: 'http://789ghi012jkl.jpg',
+      },
+    ],
+  },
+};
+
+let sampleFailResult = {
+  successCode: 0,
+  errPoint: null,
+  err: null,
+};
+
+// Send mentoring request pushing Email to mentor(receiver)
+function sendRequestEmail(req, res, receiver) {
+  let transport
+    = mailer.createTransport('smtps://yoda.mentor.lab%40gmail.com:svbootcamp@!@smtp.gmail.com');
+  let mailOptions = {
+    from: '"Yoda Service Team" <yoda.mentor.lab@gmail.com>',
+    to: receiver,
+    subject: 'New mentee needs your help!',
```

Comment:
Actually, hard coded objects like this should be defined as a constant, and declared at the top (just like sampleSuccessResult).